web/static/css/design-tokens.css

```
@@ -341,3 +597,45 @@ body { font-family: var(--sw-font-family); }
+ * Tailwind v4 is CSS-first (@import "tailwindcss"), so @theme is the v4 form of
+ * v3's `theme.extend`. Publishing the seeds and decorators here means a utility
+ * class can reach for a TOKEN (border-swd-line, text-swd-err, shadow-sw-halo)
+ * instead of an arbitrary value (border-[#3b82f6]) -- i.e. utilities cannot
+ * reintroduce the literals this PR just removed.
+ *
+ * Deliberately NOT published as a --color-* (which would mint bg-* utilities):
+ * --swd-accent. An `bg-swd-accent` utility would be an OPAQUE ACCENT FILL, the
+ * one thing the design rule forbids. The accent is reachable as a border, as
+ * ink, as the 22% tint, and as the halo -- and that is the complete list.
+ * ========================================================================= */
+@theme {
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@theme`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)